tests/integration/targets/vmware_vcenter_settings/tasks/main.yml

```diff
@@ -163,3 +163,53 @@
   assert:
     that:
       - configure_settings_idempotency_check_result.changed is sameas false
+
+- when: vcsim is not defined
+  block:
+  - name: Configure advanced settings
+    vmware_vcenter_settings:
+      hostname: "{{ vcenter_hostname }}"
+      username: "{{ vcenter_username }}"
+      password: "{{ vcenter_password }}"
+      validate_certs: false
+      database:
+        max_connections: 50
+        task_cleanup: true
+        task_retention: 180
+        event_cleanup: true
+        event_retention: 180
+      mail:
+        server: mail.example.local
+        sender: vcenter@{{ inventory_hostname }}
+      advanced_settings:
+        'config.vcls.clusters.domain-c8.enabled': 'false'
```

Comment:
This value is based on cluster MoID. I would recommend to gather info about cluster and then use that here.

```yaml
  - name: Get Cluster Managed Object ID
    vmware_cluster_info:
      cluster_name: "{{ c1 }}"
      schema: vsphere
      properties:
      - name
      - _moId
    register: r

  - set_fact:
      key: "config.vcls.clusters.{{ r.clusters[c1]['moid'] }}.enabled"
```

Reply:
> This value is based on cluster MoID. I would recommend to gather info about cluster and then use that here.

Then the vCenter only job [fails](https://35d148af1a25b437b860-7727c48abd3fb97b77ac1a581a140651.ssl.cf5.rackcdn.com/841/8bb03e0a1e7243e3640c396051dd181f3d0b35bc/check/ansible-test-cloud-integration-vcenter7_only-python36-stable210/9b38552/job-output.html#l41418).

But I don't think this is really a problem, vCenter seems to accept it no matter if a cluster witd MoID `domain-c8` exists or not. And that's the important test: Does the module configure the advanced option?